packages/block-library/src/cover/edit.js

```diff
@@ -225,25 +254,27 @@ class CoverEdit extends Component {
 			</>
 		);
 
-		if ( ! url ) {
-			const placeholderIcon = <BlockIcon icon={ icon } />;
-			const label = __( 'Cover' );
+		if ( isEditing || ! url ) {
```

Comment:
I wonder whether the `|| ! url` part is needed. 

If it is, it could cause some minor issues. For example the toolbar button for toggling editing only shows as selected using by checking the `isEditing` bool. If there's a case where `isEditing` is false, but `! url` is true, that would mean the button is presented in the wrong way.

Reply:
apparently it wasn't needed!